packages/generator-common/public-api.md

```diff
@@ -0,0 +1,6 @@
+### Generator-Common Package Public API
+
+Public API is empty => this should get reimported on generator and openapi-generator
+
+- function transpileDirectory()
+- function readCompilerOptions()
```

Comment:
Maybe keep this list empty and move to the generator packages?

Reply:
I leave a comment